engine/src/main/resources/settings.ini

```diff
@@ -4,7 +4,7 @@ fullscreen=false
 controlType=1
 sfxVol=1.0
 musicVol=0.25
-canSellEquippedItems=false
```

Comment:
Any reason for changing this?

Reply:
I changed it to true deliberately ,in-game.

Reply:
Actually, I can agree with this change - the player shouldn't be able to sell equipped items. Leave it be for now.